Cangjie5.txt

```diff
@@ -2551,6 +2551,7 @@
 𰭒	bbmmo
 𱅳	bbmmp
 𩩑	bbmmr
+𩨗	bbmms
```

Comment:
We already have
```
𩨗	bbmjs
```

The new mapping is to match the G-source glyph
![image](https://user-images.githubusercontent.com/3607926/112729645-a3295180-8f03-11eb-8b83-beb87b21c63b.png)